components/shared/Mentor/mentor-container.tsx

```diff
@@ -18,7 +18,7 @@ const MentorContainer = ({ users }: MentorContainerProps) => {
       try {
         // fetches the userDetails from db
         const response = await fetch(
-          `http://localhost:3001/userDetails?userId=${currentUser.userId}`
+          `http://localhost:3001/userDetails?id=${currentUser.id}`
         );
 
         if (!response.ok) {
```

Comment:
does it make sense "Network response was not ok"
do you have a plan ? to  handle this error message throw new Error("Network response was not ok");

Reply:
thanks, i have noted this issue. i will fix this when i completely redo the fetching implementation according to the new schema that the backend will send me/